pkg/minikube/download/binary.go

```diff
@@ -88,3 +89,22 @@ func Binary(binary, version, osName, archName, binaryURL string) (string, error)
 	}
 	return targetFilepath, nil
 }
+
+// DownloadCrictlBinary download the crictl tar archive to the cache folder and untar it
+func DownloadCrictlBinary(k8sversion string) (string, error) {
+	v, err := util.ParseKubernetesVersion(k8sversion)
+	if err != nil {
+		return "", err
+	}
+	crictlVersion := fmt.Sprintf("v%d.%d.0", v.Major, v.Minor)
+	url := fmt.Sprintf(
+		"https://github.com/kubernetes-sigs/cri-tools/releases/download/%s/crictl-%s-linux-%s.tar.gz",
+		crictlVersion, crictlVersion, runtime.GOARCH)
+	targetDir := localpath.MakeMiniPath("cache", "linux", runtime.GOARCH, k8sversion)
+	targetPath := path.Join(targetDir, "crictl")
+	if err := download(url, targetPath); err != nil {
```

Comment:
We should run `checkCache` before downloading crictl to ensure we don't download it when it already exists